internal/testutil/pkgdata.go

```diff
@@ -265,3 +265,11 @@ func MakeDeb(entries []TarEntry) ([]byte, error) {
 	}
 	return buf.Bytes(), nil
 }
+
+func MakeTestDeb(entries []TarEntry) []byte {
+	data, err := MakeDeb(entries)
+	if err != nil {
+		panic(err)
+	}
+	return data
+}
```

Comment:
Just a creative proposal to even simplify the change further, if you find it to your liking.

Inspired by:
```testutil/exec.go: FakeCommand(c *check.C, ...)```

Given this is a testutil package, facilitating tests, I would imagine we do not care about error details for ```MakeDeb```, we always simply do c.Assert(err, IsNil)``` afterwards.

Could we simply change to ```MakeDeb(c *check.C, ...)``` and include the ```c.Assert(err, IsNil)``` at the end?

Reply:
Example usage from later commits:
```
type extractTest struct {
	...
	pkgdata []byte
	...
}

var extractTests = []extractTest{{
	...
}, {
	summary: "Implicit parent directories",
	pkgdata: testutil.MakeTestDeb([]testutil.TarEntry{
		DIR(0701, "./a/"),
		DIR(0702, "./a/b/"),
		REG(0601, "./a/b/c", ""),
	}),
	...
}, {
	summary: "Implicit parent directories with different target path",
	pkgdata: testutil.MakeTestDeb([]testutil.TarEntry{
		DIR(0701, "./a/"),
		DIR(0702, "./b/"),
		REG(0601, "./b/x", "shark"),
		DIR(0703, "./c/"),
		REG(0602, "./c/y", "octopus"),
		DIR(0704, "./d/"),
	}),
	...
}, {
	...
}
```

Reply:
You are right, sorry, I missed the use case.